server.js

```diff
@@ -8,7 +8,7 @@ app.set('port', (process.env.PORT || 3000));
 var MongoClient = require('mongodb').MongoClient
 var db;
 var password = process.env.MONGO_PASSWORD;
-var mongo_connection = "mongodb://cs336:" + password + "@ds249583.mlab.com:49583/cs336";
+var mongo_connection = "mongodb://cs336:" + password + "@ds127944.mlab.com:27944/photps";
```

Comment:
I don't know which db we would want to keep... if we switch to your database, could you share the creds? or could we stick to the previous db?